docs/application/dotnet/guides/applications/nui-widget-app.md

```diff
@@ -0,0 +1,244 @@
+
+To summarize, you [create a NUI WidgetApplication](#create-a-nui-widget-application), [create a NUI WidgetView](#create-a-nui-widget-view), and use Widget to view layout on the screen.
+
+## Instance States and Events
+
+The `Widget` class provides interface for creating custom widget.
+The following table lists the main functions to manage the widget instance:
+
+**Table: Main functions**
+| Function           | Description                                                                                   |
+|--------------------|-----------------------------------------------------------------------------------------------|
+| `OnCreate()`       | This function is called after the widget instance is created.                                 |
+| `OnTerminate()`    | This function is called after the widget instance is terminated.                              |
+| `OnPause()`        | This function is called when the widget is invisible.                                         |
+| `OnResume()`       | This function is called when the widget is visible.                                           |
+| `OnResize()`       | This function is called before the widget size is changed.                                    |
+| `OnUpdate()`       | This function is called when an event for updating the widget is received.                    |
+| `SetContentInfo()` | This function sends data about current state of widget instance to a widget viewer application. If you want to save the current state of widget before deleting it, you can save it to the current status using `SetContentInfo()`.|
+
+Most of functions are used for managing widget instance life-cycle.
+
+## Prerequisites
+To enable your application to use the NUI widget functionality:
+
+To use the methods and properties of [Tizen.NUI](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.html) namespace, include it in your application:
+
+   ```csharp
+   using Tizen.NUI;
+   ```
+
+## Create a NUI Widget Application
+
+If you want to use NUI for your widget application, you must create the application with [NUI Widget Application](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html) class. 
+
+1. Define your widget application class, which is inherited from [NUIWidgetApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html) class:
+
+   ```csharp
+   class Program : NUIWidgetApplication
+   ```
+3. The widget application starts with `Main()`, which creates and initializes the application. `Run()` method of [NUIWidgetApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html) class is used to start the application event-loop.
+
+   `NUIWidgetApplication` class provides two kinds of constructors:
+   - For using the `NUIWidgetApplication(Type type)` constructor, in case the widget application's ID is the same as the application ID.
```

Comment:
Please use colon instead of period as code follows:

```suggestion
- For using the `NUIWidgetApplication(Type type)` constructor, in case the widget application's ID is the same as the application ID:
```